update to burger menu for mobile

sareg0 opened the pull request:
updated the 'burger' to look a little better on mobile... particularly iphones ;)

the other navigation items will now disappear the smaller the screen gets.

sareg0 committed:
update to burger menu for mobile

alicetragedy commented:
It does look better :)
two things I noticed though:
1) in full view, the menu is now moved almost to the middle of the page, I think it would look better if it stayed on the right side
2) when I resize my browser, at some point the menu icon jumps down (I think it's in the tablet size). I can definitely take a look at it tomorrow or on the weekend, or maybe you have some time to?

attaching screenshots.

![screen shot 2015-03-26 at 23 22 53](https://cloud.githubusercontent.com/assets/3143348/6858482/3b565662-d40f-11e4-8eda-e7ce7c1f90b0.png)

![screen shot 2015-03-26 at 23 17 55](https://cloud.githubusercontent.com/assets/3143348/6858405/90cb6b9c-d40e-11e4-919c-19d4520e695e.png)

sareg0 committed:
fix navigation for mobile

I removed a div class 'container-fluid' which was making things very strange on mobile. I did this because the gutter width didn't have a unit value. When applied as a unit value of 30px, the 'container-fluid' was creating a 15px white space gutter on either side the page, so I removed that div as it was structurally unneccessary.

sareg0 commented:
Okay, so I didn't integrate your changes @alicetragedy but found a more structurally sound way of fixing things, that made the entire page look a bit better on mobile (the text for instance now has some nice padding on mobile). Do let me know if it's all good. It seems to be working nicely for me.
